src/Resources/config/services.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    contao_community_alliance.event_dispatcher.populator:
```

Comment:
We already have the shortcut almost everywhere, this one however slipped through :smile_cat: